### ISSUE TITLE
meeting 8/16

### ISSUE THREAD

**Issue by bellecarrell** [bellecarrell/twitter_brand#29]
what i've been doing:

- remembering what i'd already done with code (get + filter user ids. no code to pull down tweets though. should be doable with twitter-search --past-tweets)
- reorganizing
- adding all current ideas to paper

what i want to do next:
- write code for data collection of user tweets (either (1) pull last 3200 + partition by pulling out timestamps on tweet object (2) start collecting sept 1 and collect 3 months (sept, oct, nov) for all users)
- at the same time, pick final 5 brands/roles (artist, writer, whatever) by beginning of september (iff need to collect via (2))
- then try out neural + interpretable classical (like in the one paper http://aclweb.org/anthology/P18-2032, https://arxiv.org/pdf/1802.06613.pdf) regression models

deadlines:
- sept 1 for finding user set (things to figure out -- 5 brands, # of users in each follower interval)
- sept 1 for having code to collect data

questions:
1.  user set -- need to validate by hand or something else? current filters -- account made after 2017, total tweets between 90 and 4450, still active -- enough? any others needed?
2. collecting user tweets -- last 3200 for our users + partition for time series analysis feasible? (3 posts/day ~~ 1000 tweets/year, so 3200 should cover 1 year? or definitely at least 3 months of activity) also, filtering non english tweets?
3. how to incorporate marketing research? most likely discuss w/ dirk when he's back from vacation
4. hypotheses for regression beyond LDA + baseline = better?

**Comment by abenton** [comment]
Belieber paper technique to identify occupations:

All role-representative users were drawn from the free public 1% sample of the Twitter Firehose, over the period 2011-2013, from the subset that selected English as their native language (85,387,204 unique users). To identify users of a particular role, we performed a case-agnostic search of variants of a single pattern: I am a(n), and I’m a(n) , where all single tokens filling the slot were taken as evidence of the author self-reporting for the given “role”. Example tweets can be seen in Table 1, examples of frequency per role in Table 2. This resulted in 63,858 unique roles identified, of which 44,260 appeared only once.1

We manually selected a set of roles for further exploration, aiming for a diverse sample across: occupation (e.g., doctor, teacher), family (mother), disposition (pessimist), religion (christian), and “followers” (belieber, directioner).2 We filtered users via language ID (Bergsma et al., 2012) to better ensure English content.
